client/src/featureform/register.py

```diff
@@ -3971,6 +4031,9 @@ def register_training_set(
         schedule: str = "",
         tags: List[str] = [],
         properties: dict = {},
+        task_id: int = 0,
+        job_id: int = 0,
+        trigger: Union[List[str], List[TriggerResource]] = [],
```

Comment:
we shouldn't set to a default list, bad python. It shouldn't cause a bug here but just something to watch out